pages/common/mongo.md

```diff
@@ -0,0 +1,19 @@
+# mongo
+
+> MongoDB shell client.
```

Comment:
Ah, that changes everything 😄 in that case, I would suggest "MongoDB interactive shell client", "Mongo REPL", or something like that. Just "shell client" makes it seem like it's a regular command-line client for the shell (whichever one you're using).